projects/core/src/lib/colors/color-palette-editor/color-palette-editor.component.html

```diff
@@ -0,0 +1,12 @@
+<button (click)="debugClick()">Print Debuginfo</button>
+<br />
+
+<br />
+<div *ngFor="let c of allColors; let i = index">
+    <geoengine-color-attribute-input [ngModel]="c" (ngModelChange)="rebuildColorMap(i, $event)"></geoengine-color-attribute-input>
+    <button mat-mini-fab (click)="removeColorTab(i)">-</button>
```

Comment:
![grafik](https://user-images.githubusercontent.com/4947061/221537687-204213dc-cb23-49b1-bf7b-08d917e9ebd8.png)

This looks kind of ugly and the button is vertically unaligned.

Reply:
Aligned it better and added some padding 7b9aacac815d6eac4398b0466d4c9b74d043a732